Helper_Scripts/rg_stage9a_parity_window.py

```
@@ -0,0 +1,657 @@
+def _write_markdown_report(path: Path, lines: Iterable[str]) -> None:
+    path.parent.mkdir(parents=True, exist_ok=True)
+    path.write_text("\n".join(lines).rstrip() + "\n", encoding="utf-8")
+
+
+def cmd_exercise(args: argparse.Namespace) -> int:
+    """Exercise all expected RG-governed endpoints to generate decisions."""
+    import subprocess
+
+    base = args.base_url.rstrip("/")
+    key = args.api_key or ""
+    headers = _headers(key, args.bearer_token)
```

Ruff (0.14.8)

Local variable `headers` is assigned to but never used

Remove assignment to unused variable `headers`

(F841)